guides/08_Migration_to_2_0.md

```diff
@@ -126,6 +126,9 @@ try {
 await audioMixController.bindAudioStream(destinationStream.stream);
 ```
 
+Additionally, the `bindAudioDevice()` will throws an error with a message `'Cannot select audio output device. This browser does not support setSinkId.'`
+if the browser does not support [`setSinkId()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/setSinkId) API. Read more about browsers supporting `setSinkId`: https://caniuse.com/?search=setSinkId
```

Comment:
'if the browser does not support setSinkId API, then...'
We are missing the 'then' part

Reply:
I think we can reword this to:
Additionally, if the browser does not support [`setSinkId()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/setSinkId) API, the `bindAudioDevice()` will throw an error with a message `'Cannot select audio output device. This browser does not support setSinkId.'` Check [caniuse.com](https://caniuse.com/?search=setSinkId) for the `setSinkId` browser support.